web/src/components/Footer.tsx

```diff
@@ -1,27 +1,38 @@
-import { TFunction } from 'i18next'
 import logo from 'web/assets/oph_fin_vaaka.png'
 import { feedbackKey } from './routes/routes'
 import { ExternalLink } from './ExternalLink'
 import { InternalLink } from './InternalLink'
 import { OPH_URL, TIETOSUOJA_SELOSTE_URL } from '../constants'
+import { useTranslation } from 'react-i18next'
 
-export const Footer = ({ t }: { t: TFunction }) => (
-  <div className="mt-5 border-t-2 border-gray-separator pb-4 pt-3">
-    <div className="flex flex-wrap md:px-10">
-      <div className="flex w-full flex-col md:w-3/12">
-        <div className="flex justify-center pb-5 pt-3">
-          <a href={OPH_URL} target="_blank" rel="noopener noreferrer" title="Opetushallituksen nettisivut">
-            <img className="h-12" src={logo} alt="Opetushallituksen logo" />
-          </a>
+type FooterProps = {
+  isPresentation?: boolean
+}
+
+export const Footer = ({ isPresentation }: FooterProps) => {
+  const { t } = useTranslation()
+  return (
+    <div className="mt-5 border-t-2 border-gray-separator pb-4 pt-3">
+      <div className="flex flex-wrap md:px-10">
+        <div className="flex w-full flex-col md:w-3/12">
+          <div className="flex justify-center pb-5 pt-3">
+            <a href={OPH_URL} target="_blank" rel="noopener noreferrer" title="Opetushallituksen nettisivut">
```

Comment:
Pitäsköhä ton title olla parempi 😂 mun laiskuudesta jäänyt toi "Opetushallituksen nettisivut"

Reply:
Meinaatko, että pitäis olla lokalisoitu? Mä taisin lisätä ton "nettisivut" osan tohon.

Reply:
Näist saavutettavuusjutuista on oma muistutuslappu mirossa, varmaan siinä tulee sit hoidettuu nää.